From b0c8d65a493ce5230b36014a788d82b15a9a0ad6 Mon Sep 17 00:00:00 2001
From: "Jose M. Guisado Gomez" <guigom@riseup.net>
Date: Sun, 26 Jul 2020 10:16:23 +0200
Subject: [PATCH] nftables plugin: fix setup_rule call

unbreak setup_rule call since it does not need a fourth argument
---
 src/nftables.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/nftables.c b/src/nftables.c
index 1a6236e7e6..7ba38b4c1a 100644
--- a/src/nftables.c
+++ b/src/nftables.c
@@ -209,7 +209,7 @@ int nl_match_rules() {
     nlh = nftnl_rule_nlmsg_build_hdr(buf, NFT_MSG_GETRULE, family, NLM_F_DUMP,
                                      seq);
 
-    r = setup_rule(family, table, chain, NULL);
+    r = setup_rule(family, table, chain);
 
     if (mnl_socket_sendto(nl, nlh, nlh->nlmsg_len) < 0) {
       ERROR("nftables plugin: Error sending to mnl socket");
